R/cube-result.R

```diff
@@ -294,6 +297,14 @@ cubeMarginTable <- function (x, margin=NULL, measure=1) {
     ## for which that would do the wrong thing.
     keep.these <- evalUseNA(mt, dims[mt_margins], x@useNA)
     out <- subsetCubeArray(mt, keep.these)
+
+    # only attempt to apply a transform if the margin is 1 rows for now.
+    if (!is.null(margin) && margin == 1) {
+        margin_trans <- dims[[mapped_margins]]$references$view$transform
+        if (!is.null(margin_trans)) {
```

Comment:
Seems you already do this check for existence of transforms (more robustly, in fact) in `applyTransforms`: https://github.com/Crunch-io/rcrunch/pull/139/files#diff-fc6c252cf7a4ec51064a3ca7fde28541R91